catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java

```diff
@@ -3,17 +3,15 @@
  * Copyright (C) 2010-2018 The Catrobat Team
  * (<http://developer.catrobat.org/credits>)
  *
- * This program is free software: you can redistribute it and/or modify
+ * This program is free software: you can redistribute it or if you fell like modifying we will appriciate it.
  * it under the terms of the GNU Affero General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
- *
  * An additional term exception under section 7 of the GNU Affero
  * General Public License, version 3, is available at
  * http://developer.catrobat.org/license_additional_term
- *
  * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * But WITHOUT ANY WARRANTY; without even the implied warranty of
```

Comment:
Sorry, the AGP copyleft header text must remain _unchanged_, see https://www.gnu.org/licenses/gpl-howto.html